.circleci/config.yml

```diff
@@ -41,14 +41,14 @@ jobs:
       # Restore bundle cache
       - restore_cache:
           keys:
-            - zync-bundle-v2-{{ arch }}-{{ checksum "Gemfile.lock" }}
-            - zync-bundle-v2-{{ arch }}-{{ .Branch }}
-            - zync-branch-v2-{{ arch }}-master
+            - zync-bundle-v2-{{ .Environment.CACHE_VERSION }}-{{ arch }}-{{ checksum "Gemfile.lock" }}
```

Comment:
This was also just to try things out. It's similar to what we do in portal. If it bothers here, I can remove.